platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/SwirldStateManagerImpl.java

```diff
@@ -38,18 +37,18 @@
 import org.apache.logging.log4j.Logger;
 
 /**
- * <p>Manages all interactions with the state object required by {@link SwirldState2}.</p>
+ * <p>Manages all interactions with the state object required by {@link SwirldState}.</p>
  *
  * <p>Two threads interact with states in this class: pre-consensus event handler and consensus event handler.
  * Transactions are submitted by a different thread. Other threads can access the states by calling
  * {@link #getCurrentSwirldState()} and {@link #getConsensusState()}. Sync threads access state to check if there is
  * an active freeze period. Careful attention must be paid to changes in this class regarding locking and
  * synchronization in this class and its utility classes.</p>
  */
-public class SwirldStateManagerDouble implements SwirldStateManager {
+public class SwirldStateManagerImpl implements SwirldStateManager {
```

Comment:
Theoretically with the renaming of this class, the new `PlatformThreads` drawio diagram which just merged into develop would need to be updated to match.